dotnet/samples/README.md

```diff
@@ -1 +1,121 @@
-placeholder
\ No newline at end of file
+# Samples
+
+In this folder you will find various examples showcasing the different capabilities of the .NET Teams AI Library. Here are the general instructions for setting up a sample: 
+
+## Prerequisites
+
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [Azure OpenAI](https://aka.ms/oai/access) resource or an account with [OpenAI](https://platform.openai.com).
+
+### Consuming the latest version of the library (preview only) 
+
+> **NOTE:**  As the library has not been published to NuGet's public registry, please complete the following steps to be able to use the samples. Otherwise, the samples will not work.
+
+The development versions of the library might not be available on NuGet's public registry. Follow the instructions below to generate the `.nukpg` NuGet package file of the library to locally consume within the sample:
+
+1. Clone the repository:
+
+`git clone https://github.com/Microsoft/teams-ai.git`
+
+2. Checkout the `DOTNET` branch.
+
+3. Navigate to the `teams-ai/dotnet/packages/Microsoft.TeamsAI` folder.
+
+4. Run `dotnet pack` in terminal.
+
+5. You should then see the following output: 
+
+`Successfully created package "C:...\teams-ai\dotnet\packages\Microsoft.TeamsAI\Microsoft.TeamsAI\bin\Debug\Microsoft.TeamsAI.1.0.0.nupkg"`
+
+6. Move the `Microsoft.TeamsAI.1.0.0.nupkg` to the `LocalPkg/` folder within the sample folder.
+
+7. Now you can install the `Microsoft.TeamsAI` package from the Visual Studio's `Manage NuGet Packages` flow or by running `dotnet add Microsoft.TeamsAI`.
+
+Now can you proceed to setting up the sample.
+
+## Setting up a sample
+
+1. Clone the repository:
+
+`git clone https://github.com/Microsoft/teams-ai.git`
+
+2. Navigate to the `teams-ai/dotnet/samples` folder, pick a sample (ex. `01.a.echoBot`) and open the `.sln` file. 
+
+By this point you should have your sample open in your IDE of choice.
+
+There are a few ways to get the application up and running. The latest way is using Teams toolkit with Visual Studio. However you can also set it up manually. You can find instructions for both below:
+
+### Using Teams Toolkit for Visual Studio
+
+#### Additional Prerequisites
+
+- [Visual Studio 2022 v17.3.0+ (Preview version)](https://visualstudio.microsoft.com/vs/preview/)
```

Comment:
[Visual Studio 2022 **v17.7 Preview 3**]?

The [release note](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview#17.7.0-pre.3.0) calls it "Visual Studio 2022 version 17.7 Preview 3". Or in the installer it's shown as "17.7.0 Preview 3.0".

Reply:
Oops, fixed that.